src/afw.jl

```diff
@@ -93,7 +97,7 @@ function away_frank_wolfe(
         )
         grad_type = typeof(gradient)
         println(
-            "GRADIENTTYPE: $grad_type LAZY: $lazy K: $K MOMENTUM: $momentum AWAYSTEPS: $awaySteps",
+            "GRADIENTTYPE: $grad_type LAZY: $lazy K: $K MOMENTUM: $momentum away_steps: $away_steps",
```

Comment:
```suggestion
            "GRADIENTTYPE: $grad_type LAZY: $lazy K: $K MOMENTUM: $momentum AWAYSTEPS: $away_steps",
```

for consistency in reporting all parameters are in CAPITALS only.